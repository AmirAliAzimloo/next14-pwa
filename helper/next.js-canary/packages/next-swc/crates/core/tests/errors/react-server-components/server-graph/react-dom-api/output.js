import { findDOMNode, flushSync, unstable_batchedUpdates } from 'react-dom';
import { useFormStatus, useFormState } from 'react-dom';
export default function() {
    return null;
}
