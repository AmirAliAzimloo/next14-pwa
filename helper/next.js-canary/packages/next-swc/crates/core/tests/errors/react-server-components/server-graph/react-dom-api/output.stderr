
  x NEXT_RSC_ERR_REACT_API: findDOMNode
   ,-[input.js:1:1]
 1 | import { findDOMNode, flushSync, unstable_batchedUpdates } from 'react-dom'
   :          ^^^^^^^^^^^
   `----

  x NEXT_RSC_ERR_REACT_API: flushSync
   ,-[input.js:1:1]
 1 | import { findDOMNode, flushSync, unstable_batchedUpdates } from 'react-dom'
   :                       ^^^^^^^^^
   `----

  x NEXT_RSC_ERR_REACT_API: unstable_batchedUpdates
   ,-[input.js:1:1]
 1 | import { findDOMNode, flushSync, unstable_batchedUpdates } from 'react-dom'
   :                                  ^^^^^^^^^^^^^^^^^^^^^^^
   `----

  x NEXT_RSC_ERR_REACT_API: useFormStatus
   ,-[input.js:3:1]
 3 | import {
 4 |   useFormStatus,
   :   ^^^^^^^^^^^^^^^^^^^^^^^^^^
 5 |   useFormState,
   `----

  x NEXT_RSC_ERR_REACT_API: useFormState
   ,-[input.js:4:1]
 4 |   useFormStatus,
 5 |   useFormState,
   :   ^^^^^^^^^^^^^^^^^^^^^^^^^
 6 | } from 'react-dom'
   `----
